benchmarks/dynamo/common.py

```diff
@@ -1192,10 +1188,22 @@ def load(cls, model, example_inputs, eager_forward):
         )
 
 
-def export_aot_inductor(model, example_inputs, eager_forward):
-    module, exported, output_spec = AOTInductorModelCache.load(
-        model, example_inputs, eager_forward
-    )
+def export(model, example_inputs):
+    example_args, example_kwargs = _normalize_bench_inputs(example_inputs)
+    example_outputs = model(*example_args, **example_kwargs)
+    _register_dataclass_output_as_pytree(example_outputs)
+
+    ep = torch.export.export(model, example_args, example_kwargs)
+
+    def opt_export(_, example_inputs):
+        example_args, example_kwargs = _normalize_bench_inputs(example_inputs)
+        return ep(*example_args, **example_kwargs)
```

Comment:
`pytree.tree_unflatten(output_tensors, output_spec)`?

Reply:
Not needed, exported program will do flattening + unflattening.